code-challenges.test.js

```diff
@@ -23,10 +25,47 @@ var num3 = -7
 // Expected output: "-7 is not divisible by three"
 
 
+describe("divByThree", () => {
+    it("returns whether a number is divisible by three or not", () => {
+        //expect the outcome of the function to equal the desired string
+        expect(divByThree(num1)).toEqual("15 is divisible by three")
+        expect(divByThree(num2)).toEqual("0 is divisible by three")
+        expect(divByThree(num3)).toEqual("-7 is not divisible by three")
+    })
+})
 
-// b) Create the function that makes the test pass.
+// got "RED"
 
+// b) Create the function that makes the test pass.
 
+// // declare function divByThree that takes a number as an argument
+// const divByThree = (num) => {
+//     // if the number is divisible by three its remainder will be 0
+//     if (num % 3 === 0) {
+//         // and it will return that it is divisible by three
+//         return `${num} is divisible by three`
+//         // otherwise
+//     } else {
+//         // return that it is not divisble by three
+//         return `${num} is not divisible by three`
+//     }
+// }
+
+// got red saying that  
+//      "● Test suite failed to run
+//               Your test suite must contain at least one test."
+//
+// Checked initial repo on github and found a difference in code on lines 4 and 5. The code that is there was not in the github repo and once commented out allowed my test to return green. Am curious as to why it happened.
+
```

Comment:
VSCode adds unnecessary imports to the code file which prevent the tests from passing. they just need to be removed.